tket2-hseries/src/lazify_measure.rs

```diff
@@ -101,7 +101,7 @@ lazy_static! {
     static ref MEASURE_READ_HUGR: Hugr = {
         let mut builder = DFGBuilder::new(Signature::new(qb_t(), vec![qb_t(), bool_t()])).unwrap();
         let [qb] = builder.input_wires_arr();
-        let [qb, lazy_r] = builder.add_lazy_measure(qb).unwrap();
+        let [qb, lazy_r] = builder.add_lazy_measure_reset(qb).unwrap();
```

Comment:
does this mean all measurements are re-written to ones that also reset? A normal `Measure` should become a `LazyMeasure` no?

Reply:
Ah this appears to be addressed by the follow-up PR